src/App.js

```diff
@@ -471,8 +468,8 @@ function App() {
             </li>
           </span>
 
-          {selectedSeason && !selectedArtist
-            ? renderHymn(hymns, artistData)
+          {selectedSeason
+            ? renderHymn(hymns, selectedArtist)
```

Comment:
pass the current selected artist instead of the whole artist list (this fixes the artist image not loading in hymn page)